File: HealthSPAR-main/booked-appointments/src/main/java/com/stackroute/bookedappointments/model/Appointment.java
package com.stackroute.bookedappointments.model;

import com.fasterxml.jackson.annotation.JsonFormat;
import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Table(name = "appointments_table")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Appointment {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int appointmentId;
    private String patientId;
    private Long hospitalId;
    private String treatmentType;
    private LocalDateTime dateTime;
    private String message;
    private String status;          //reschedule/cancel/book
    private String department;
    private String doctor;
    private String patientName;
    private String email;
    private String phoneNumber;
}

File: HealthSPAR-main/product-webapp/healthspar-app/src/app/model/appointment.ts
export interface Appointment {
    appointmentId: number;
    patientId: string;
    hospitalId: number;
    treatmentType: string;
    dateTime: Date;
    message: string;
    status: string;
    department: string;
    doctor: string;
    patientName: string;
    email: string;
    phoneNumber: string;
}